Source/WebCore/platform/graphics/cocoa/WebCoreDecompressionSession.mm

```diff
@@ -404,6 +404,7 @@
 
     --m_framesBeingDecoded;
     m_isDecodingSample = false;
+    m_lastDecodingError = noErr;
```

Comment:
Why is it needed?

Reply:
this is to prevent the error callback to be called over and over for all pending decoding, even if the following decodes are successful.